serverless/src/submitSwodlrOrder/retrieveCMRGranules.js

```diff
@@ -0,0 +1,47 @@
+import axios from 'axios'
+
+import { stringify } from 'qs'
```

Comment:
What is this stringify? Is built in javascript version not the same? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

Reply:
It formats the object in a specific way (with brackets for example)

Reply:
Is this different from JSON Stringify?